packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/ItemCharts/StockDistribution.tsx

```diff
@@ -112,18 +114,56 @@ const MonthlyBar = ({
   );
 };
 
+const CalculationError = ({
+  isAmcZero,
+  isSohAndQtyZero,
+}: {
+  isAmcZero?: boolean;
+  isSohAndQtyZero?: boolean;
+}) => {
+  const t = useTranslation('replenishment');
+  const detail = isAmcZero
```

Comment:
I really don't like nested ternary operators. but it seemed like the best option..

Reply:
Yeah, in principle I don't like them either, but the way VSCode formats them makes them pretty easy to follow 👌